exporters/variables-scss/src/generators/cssObjectGenerator.ts

```diff
@@ -86,8 +126,20 @@ export const colorGroupsReducer = (accumulatedColorKeys: { [key: string]: string
   [parseGroupName(currentColorKey)]: currentColorKey,
 });
 
+export const typographyGroupReducer = (
```

Comment:
I think it's starting to get a bit large. I'd rather move it into separate files. But we can do it as part of some "polishing" issue @literat ?

Reply:
let's do the refactoring after this issue with other things ready to refactor 😄